src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -465,12 +497,8 @@ def getdata(urls, ckey, cert, headers=None, options=None, num_conn=50, cookie=No
         while True:
             num_q, ok_list, err_list = mcurl.info_read()
             for curl in ok_list:
-                if sys.version.startswith('3.'):
-                    hdrs = curl.hbuf.getvalue().decode('utf-8')
-                    data = curl.bbuf.getvalue().decode('utf-8')
-                else:
-                    hdrs = curl.hbuf.getvalue()
-                    data = curl.bbuf.getvalue()
+                hdrs = curl.hbuf.getvalue().decode('utf-8')
```

Comment:
Alan, please be clear, do you want to replace ALL occurrences where we call `buf.getvalue.decode('utf-8')` or this particular case. If former, it will bring much more changes (I don't mind but that was not the intention of this PR). If later it will not be consistent with the rest of the code.